package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nevermined-io/nevermined-sdk-js",
-  "version": "1.0.0-rc10",
+  "version": "1.0.0-rc11",
```

Comment:
currently we have version 1.0.0-rc12 from this branch https://github.com/nevermined-io/sdk-js/pull/449